src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    passwords from `ssl_password_file` one by one in the order they are written.
+    - If `ssl_password` and all passwords from `ssl_password_file` are
+    incorrect, or none of them is provided, Tarantool treats the private key as
+    unencrypted.
+]])
+
+M['<uri>.params.ssl_password_file'] = format_text([[
+    (Optional) A text file with one or more passwords for encrypted private
+    SSL keys provided using `ssl_key_file` (each on a separate line).
+    Alternatively, the password can be provided in `ssl_password`.
+]])
+
+M['<uri>.params.transport'] = format_text([[
+    Allows you to enable traffic encryption for client-server communications
+    over binary connections. In a Tarantool cluster, one instance might act
+    as the server that accepts connections from other instances and the client
+    that connects to other instances.
+
+    `<uri>.params.transport` accepts one of the following values:
+
+    - `plain` (default): turn off traffic encryption,
+    - `ssl`: encrypt traffic by using the TLS 1.2 protocol (EE only).
+]])
+
+-- }}} <uri>.params configuration
+
+-- {{{ app configuration
+
+M['app'] = format_text([[
+    Using Tarantool as an application server, you can run your own Lua
+    applications. In the `app` section, you can load the application and
+    provide an application configuration in the `app.cfg` section.
+]])
+
+M['app.cfg'] = format_text([[
+    A configuration of the application loaded using `app.file` or `app.module`.
+]])
+
+M['app.cfg.*'] = format_text([[
+    Specify `file` (a path to a Lua file to load an application from) or
+    `module` (a Lua module to load an application from).
+]])
+
+M['app.file'] = 'A path to a Lua file to load an application from.'
+
+M['app.module'] = 'A Lua module to load an application from.'
+
+-- }}} app configuration
+
+-- {{{ audit_log configuration
+
+M['audit_log'] = format_text([[
+    The `audit_log` section defines configuration parameters related to
+    audit logging.
+]])
+
+M['audit_log.extract_key'] = format_text([[
+    If set to `true`, the audit subsystem extracts and prints only the primary
+    key instead of full tuples in DML events (`space_insert`, `space_replace`,
+    `space_delete`). Otherwise, full tuples are logged. The option may be
+    useful in case tuples are big.
+]])
+
+M['audit_log.file'] = format_text([[
+    Specify a file for the audit log destination. You can set the `file` type
+    using the audit_log.to option. If you write logs to a file, Tarantool
+    reopens the audit log at SIGHUP.
+]])
+
+M['audit_log.filter'] = format_text([[
+    Enable logging for a specified subset of audit events. This option accepts
+    the following values:
+
+    - Event names (for example, password_change). For details, see Audit log
+    events.
+    - Event groups (for example, audit). For details, see Event groups.
+
+    The option contains either one value from Possible values section or a
+    combination of them.
```

Comment:
> For details, see Audit log events.

> For details, see Event groups.

> value from Possible values section

It says nothing without links to these sections in the documentation. I would just eliminate it from here if we can't add the links.

Reply:
Moreover, we have the `allowed_values` annotation and possible values can be suggested by the editor using the JSON Schema.

Reply:
You are right, we have `allowed_values`, so we can simplify the description. Maybe even mention them.
```lua
I['audit_log.filter.*'] = format_text([[
    Specify a subset of audit events to log by providing a value
    from the allowed list of events or groups.
]])
```

Reply:
Maybe like this?
```lua
I['audit_log.pipe'] = format_text([[
    Start a program and write logs to its standard input (`stdin`). To send
    logs to a program's standard input, you need to set `audit_log.to` to
    `pipe`. If log is a program, its pid is stored in the `audit.pid` field.
]])
```